documentation/index.html.md

```diff
@@ -43,15 +48,29 @@ The epicenter.js library is a set of services (adapters) and managers to help st
 <a name="include"></a>
 ####Including
 
-The epicenter.js library is available from our tools: <a href="https://forio.com/tools/js-libs/1.3.0/epicenter.min.js" target="_blank">https://forio.com/tools/js-libs/1.3.0/epicenter.min.js</a>. To use it in your project, simply add
+**Epicenter.js**
+
+The epicenter.js library is available from our tools: <a href="https://forio.com/tools/js-libs/1.4.0/epicenter.min.js" target="_blank">https://forio.com/tools/js-libs/1.4.0/epicenter.min.js</a>. To use it in your project, simply add
 
-    <script src="https://forio.com/tools/js-libs/1.3.0/epicenter.min.js"></script>
+    <script src="https://forio.com/tools/js-libs/1.4.0/epicenter.min.js"></script>
 
 into any of your [interface](../creating_your_interface/) files (e.g. .html and .js files).
 
+**Dependencies**
+
 The epicenter.js library depends on jQuery, so you'll also need to download jQuery for yourself, or use a hosted version. To use a hosted version, add
 
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.2/jquery.min.js"></script>
+
+**Extensions**
+
+The Epicenter platform provides a push channel, which allows you to publish and subscribe to messages within a [project](../glossary/#projects), [group](../glossary/#groups), or [multiplayer world](../glossary/#world). There are two main use cases for the channel: event notifications and chat messages.
+
+If you are planning to use channels and epicenter.js in project, you'll also need to include the epicenter-multiplayer-dependencies.js library. This is available from our tools: <a href="https://forio.com/tools/js-libs/1.4.0/epicenter-multiplayer-dependencies.js" target="_blank">https://forio.com/tools/js-libs/1.4.0/epicenter-multiplayer-dependencies.js</a>. To use it in your project, simply add
```

Comment:
Instead of mentioning channels, let's just say "If you're building a multiplayer game, you'll need to include.."